readme-update (#198)
newrelic/newrelic-observability-packs: README.md

### 💽 Data Sources

[Data Sources](https://developer.newrelic.com/collect-data/collect-data-from-any-source/) are the many sources of data you can setup with New Relic to collect and monitor data.

### 🔮 Future

Our road map for quick starts includes the addition of [Synthetics monitoring](https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring/), [NerdPacks](https://developer.newrelic.com/build-apps/) and [Flex configurations](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/flex-integration-tool-build-your-own-integration/).

## Getting Started

Contribute your own quick start to the New Relic One catalog by following the steps below:

1. [Fork the Github repository](https://help.github.com/en/github/getting-started-with-github/fork-a-repo#fork-an-example-repository)

2. [Clone your own repository to your local machine](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository)

> 3. Copy the `_template` directory and its content to a new directory within the `packs` folder. Choose a name which identifies the purpose of your Quick start, such as `rabbitmq`, `apm-errors`, `sre`, or `aws-s3`

Reviewer: just changing to small case to keep it consistent
```suggestion
3. Copy the `_template` directory and its content to a new directory within the `packs` folder. Choose a name which identifies the purpose of your quick start, such as `rabbitmq`, `apm-errors`, `sre`, or `aws-s3`
```

Reviewer 2: ```suggestion
3. Copy the `_template` directory and its content to a new directory within the `packs` folder. Choose a name which identifies the purpose of your quickstart, such as `rabbitmq`, `apm-errors`, `sre`, or `aws-s3`
```

> 4. In your new directory, you'll find the following folders: `dashboards`, `alerts`, `instrumentation`. Each folder contains a template or template directories that you can use to create entities for your Quick start.

Reviewer: Case again, also do we want to reference `instrumentation` yet when it's not quite a thing? 🤷 
```suggestion
4. In your new directory, you'll find the following folders: `dashboards`, `alerts`, `instrumentation`. Each folder contains a template or template directories that you can use to create entities for your quick start.
```

Reviewer 2: ```suggestion
4. In your new directory, you'll find the following folders: `dashboards`, `alerts`, `instrumentation`. Each folder contains a template or template directories that you can use to create entities for your quickstart.
```

    This process is similar for all other entity directories. Also, if you don't want to create entities for a given type, delete the corresponding directory.

> 5. In your pack's root directory, you'll find a `config.yml` file where you can configure your quickstart. Refer to our Contributing Guide(./CONTRIBUTING.md) for more details on Quick Start configurations.

Reviewer 2: ```suggestion
5. In your pack's root directory, you'll find a `config.yml` file where you can configure your quickstart. Refer to our Contributing Guide(./CONTRIBUTING.md) for more details on quickstart configurations.
```

> 5. In your pack's root directory, you'll find a `config.yml` file where you can configure your Quick start. Refer to our Contributing Guide(./CONTRIBUTING.md) for more details on Quick Start configurations.

Reviewer: casing
```suggestion
5. In your pack's root directory, you'll find a `config.yml` file where you can configure your quick start. Refer to our Contributing Guide(./CONTRIBUTING.md) for more details on quick start configurations.
```

Reviewer 2: ```suggestion
5. In your pack's root directory, you'll find a `config.yml` file where you can configure your quickstart. Refer to our Contributing Guide(./CONTRIBUTING.md) for more details on quickstart configurations.
```
